test/keep_going/BUILD

```diff
@@ -0,0 +1,9 @@
+subinclude("//test/build_defs")
+
+please_repo_e2e_test(
+    name = "build_all",
+    expected_failure = True,
```

Comment:
It would be good if we could test that we built fail, fail2, pass, dep_pass and fail_test rather than just we failed at some point